### DISCUSSION TITLE
Importing from stable should only happen via `ScreenImportFromStable`

### DISCUSSION THREAD

**Discussion by LittleEndu** [ppy/osu#21908] [Ideas] [answered]
With the introduction of hard links, importing from stable is starting to get a bit taunting for the user. I suggest the removal of "import X from stable" buttons and adding a single "import content from stable" button that opens `ScreenImportFromStable`

 * Would address https://github.com/ppy/osu/discussions/21828

**Comment by bdach** [comment]
Hard links are always used if available. No matter which method is used.

**Reply by LittleEndu** [reply]
Yes, but the user has no way to confirm it in-game. `ScreenImportFromStable` has different text depending on whether hard links are used or not. `ScreenImportFromStable` also prompts to change lazer storage location if hard links are supported by OS but can't be used for some reason. In my opinion the user experience is just better in every way except having to click more buttons.

**Comment by peppy** [comment]
All the buttons at the bottom of settings are temporary (before they had better ways to be accessed). These can be removed or moved to debug-only, I think.

**Reply by frenzibyte** [reply]
Debug-only sounds better, as I would like to test certain components as I perform an immediate import of a set of resources.

**Comment by frenzibyte** [comment, marked as answer]
https://github.com/ppy/osu/pull/22104

**Reply by FriendsNone** [reply]
I'm not sure how I feel about this change, now that 2023.121.0 is released. I use the import buttons (mainly beatmaps and scores) as a way to sync my progress from stable. I know that the functionality is still there but it's a little inconvenient to go through the first-time setup just to do it.

A better implementation (in my opinion) would be just a button to show `ScreenImportFromStable` in the settings panel or possibly automatically import from stable on launch (with a toggle to disable it).

**Reply by frenzibyte** [reply]
Please open a new discussion thread for more feedback on the matter. This one is no longer valid to discuss at.
